packages/web3-hooks/base/src/useFiatCurrencyRate.ts

```diff
@@ -1,12 +1,38 @@
 import { FiatCurrencyRate } from '@masknet/web3-providers'
 import { useQuery } from '@tanstack/react-query'
 import { useCurrencyType } from './useCurrencyType.js'
+import { CurrencyType } from '@masknet/web3-shared-base'
+import { pick } from 'lodash-es'
+
+const STORE_KEY = '@mask/fiat-currency-rates'
+const getData = () => {
+    const raw = localStorage.getItem(STORE_KEY)
+    if (!raw) return undefined
+    try {
+        return JSON.parse(raw) as Record<string, number>
+    } catch {
+        return undefined
+    }
+}
+
+const setData = (data: Record<string, number>) => {
+    localStorage.setItem(STORE_KEY, JSON.stringify(data))
```

Comment:
Use a settings?

Reply:
No need to rely on background service.

Reply:
if you use this on twitter.com, it will store data on twitter.com. please be aware.

Reply:
I think it's fine.